paris/build.gradle

```diff
@@ -28,9 +29,7 @@ dependencies {
     // "Warning: unknown enum constant AnnotationTarget.FIELD"
     api deps.kotlin
 
-    compileOnly project(':paris-processor')
```

Comment:
I was getting a gradle error about this once I switched to using kapt here. The whole project still builds fine when I don't include the processor though... are you sure it is needed?

Reply:
No I'm not sure. I don't remember why I added it. If everything builds and all the tests pass then it should be fine! I can't think of why we'd need it at compile time.